.devcontainer/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG IMAGE=bullseye
+FROM --platform=amd64 mcr.microsoft.com/devcontainers/${IMAGE}
+RUN export DEBIAN_FRONTEND=noninteractive \
+     && apt-get update && apt-get install -y xdg-utils \
+     && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+RUN curl -fsSL https://aka.ms/install-azd.sh | bash
```

Comment:
do we need to enable the alpha feature here as well like what you did here https://github.com/Azure-Samples/ASA-Samples-Event-Driven-Application/pull/3/files#diff-29029d1adf1f02a4a4a6eb77699dcfc8b7547f11174bb70b8ee22072b3ef1b9fR15

Reply:
Thanks for catching this, I would create another PR to fix